src/ui/hooks/useForegroundService.ts

```diff
@@ -17,12 +16,11 @@ export const useForegroundService = (): void => {
 
     // Send the sync url to bugsnag so we have more context for where an error is occurring
     Bugsnag.addMetadata('sync', 'serverUrl', serverUrl);
+    Scheduler.updateStatus('Running...');
```

Comment:
this changes the text from 'starting' to 'Running...' so that you know if the app has started and is running. more information than that is probably not helpful